libraries/types/vote/src/vote.cpp

```diff
@@ -58,9 +58,12 @@ uint64_t VrfPbftSortition::getBinominalDistribution(uint64_t stake, double dpos_
   return stake;
 }
 
-uint64_t VrfPbftSortition::getBinominalDistribution(uint64_t stake, double dpos_total_votes_count,
-                                                    double threshold) const {
-  return VrfPbftSortition::getBinominalDistribution(stake, dpos_total_votes_count, threshold, output);
+uint64_t VrfPbftSortition::calculateWeight(uint64_t stake, double dpos_total_votes_count, double threshold,
+                                           const public_t& address) const {
+  // Also hash in the address. This is necessary to decorrelate the selection of different accounts that have the same
+  // VRF key.
+  HashableVrf hash_vrf(output, address);
```

Comment:
How does this work when output is not declared here nor passed as argument ?

Reply:
output is member of class/struct

Reply:
we could fix things like this if we touch it ? VrfPbftSortition should be class, not struct and all members should have `_`

Reply:
This output initially always confused me as well specially since it is a member from the base class. We should definitely add "_". Maybe also put VrfPbftSortition class into a separate file?

Reply:
sure, I will refactor the while thing